lib/app.js

```diff
@@ -30,7 +30,13 @@ app.get('/api/test', (req, res) => {
 
 app.get('/characters', async(req, res) => {
   try {
-    const data = await client.query('SELECT * from characters');
+    const data = await client.query(` SELECT characters.id,
+        characters.name,
+        characters.bad, 
+        characters.species_id
```

Comment:
since you're not returning the breed name, this join isn't benefitting your right now. You'll want to also add `breeds.name` to your select so that you get the benefit of the join. Since you also have a `name` in the characters column, you may need to add an alias:
`breeds.name as breed_name`